g11n-ws/modules/md-service-i18n-l3/src/main/java/com/vmware/vip/core/messages/service/product/ProductService.java

```diff
@@ -61,8 +61,7 @@
 				supportedLocaleList.remove(ConstantsKeys.LATEST);
 			}
 		} catch (DataException e) {
-			throw new L3APIException(ConstantsKeys.FATA_ERROR
```

Comment:
why remove FATAL error?

Reply:
Because it's not a fatal error as it can't detect if the bundle is lost or not.

Reply:
Especially for new product.